scripts/ag

```diff
@@ -54,5 +54,30 @@ def persample_json(table, output):
             fp.write(json.dumps(tree, indent=2))
 
 
+@ag.command()
+@click.argument('accession-metadata', required=True, nargs=-1)
+def sample_to_accession(accession_metadata):
+    """Dump {accession: [sample]} and {sample: accession}"""
```

Comment:
Isn't that one of the normal notations to see? Like `[int]` implies `list` of `int`. Can change if you'd like tho